### PR TITLE
Fixes to page titles in generated docs

### PR DESCRIPTION
**Pull request by adam-cowley**
I've changed the page title levels so the generated docs don't have Untitled in the page title.

Could you please double check that the content is still correct?

I've added a new npm command for previewing the docs - they will rebuild as you make changes so you can view them at localhost:8000

```
npm start
```

### CODE REVIEW

**Comment by adam-cowley** [comment]
The streams.adoc file was causing warnings in the build process that will break future builds of the docs.  It doesn't seem to be referenced anywhere so I'm assuming it is OK to remove.  Could you please confirm?  

Here is the contents:

```adoc
include::producer.adoc[]

include::consumer.adoc[]

include::procedures.adoc[]
```

**Review by conker84 — APPROVED** [review]

File: doc/docs/modules/ROOT/pages/producer.adoc [hunk 1]
```diff
@@ -1,38 +1,38 @@
+= Source: Neo4j -> Kafka
```
Comment (mroiter-larus) [inline doc/docs/modules/ROOT/pages/producer.adoc:1]: Does this title needs the prefix "Neo4j Streams -" like the other two chapters about Sink and Procedures modules?

Comment (adam-cowley) [reply doc/docs/modules/ROOT/pages/producer.adoc:1]: No it shouldn't need the prefix - having the prefix doesn't really add any value to the page

**Review by mroiter-larus — APPROVED** [review]

**Review by mroiter-larus — APPROVED** [review]

**Review by mroiter-larus — APPROVED** [review]

**Comment by mroiter-larus** [comment]
@adam-cowley LGTM

**Pull request merged**